inference-engine/src/transformations/src/transformations/serialize.cpp

```diff
@@ -526,8 +539,8 @@ pass::Serialize::Serialize(const std::string& xmlPath,
                            const std::string& binPath,
                            pass::Serialize::Version version,
                            std::map<std::string, OpSet> custom_opsets)
-    : m_xmlPath{xmlPath}
-    , m_binPath{bestBinPath(xmlPath, binPath)}
+    : m_xmlPath{valid_xml_path(xmlPath)}
+    , m_binPath{provide_bin_path(xmlPath, binPath)}
```

Comment:
@pelszkow we depend here on initialization order (valid_xml_path should be called before provide_bin_path). Is this order defined for C++ 11 ?

Reply:
In general, class members should be initialized based on order in class definition (https://en.cppreference.com/w/cpp/language/constructor#Initialization_order -> 4 ). If you run GCC with `-Wall` flag you will get warning [https://godbolt.org/z/Y9Erdz]. 
If you want to have order more explicit in our sources we should drop `const` in class members definitions and assign `m_binPath` in the body of the constructor.
@jdanieck , which approach do you prefer?

Reply:
Thanks for explanation. Let's leave it as is, I prefer to have it const.